nwb.config.js

```diff
@@ -0,0 +1,23 @@
+const browsers = require('./test/browsers');
```

Comment:
NWB is pretty well documented. All people need to worry about, really, is writing code. I'm not too worried about this. If people have questions, they can refer to those docs or ask questions, but it's less config and stuff in the way to worry about in the end for everyone.